planner_id: RRTConnectkConfigDefault
    #RRTConnectkConfigDefault 
    #RRTstarkConfigDefault
planning_time: 1.0        #[seconds
num_planning_attempts: 10 #[number]